pagefaultgames/pokerogue
# [Localization] Add Korea type icons

PR opened on behalf of alakh8187 on Discord Korean tranlsator.

## What are the changes?
Korean incons for types

## Why am I doing these changes?
To give korean players proper localized icos

## What did change?
Addidion of icons in Korean

### Screenshots/Videos
![image](https://github.com/pagefaultgames/pokerogue/assets/2070109/62a0aa32-83f4-4d80-935f-72ec6310db84)
![image](https://github.com/pagefaultgames/pokerogue/assets/2070109/6f3e4c48-a60c-4a77-9c4e-43ce985807f6)
![image](https://github.com/pagefaultgames/pokerogue/assets/2070109/757220f3-8a56-4b56-8657-6d26ec66b243)

## How to test the changes?
Just check anywere type icones are supposed to be displayed (starter selection, move selection in battle, move selection in summary, Pokémon type in summary...)

## Checklist
